src/figuresClasses.ts

```diff
@@ -1,11 +1,66 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape = 'triangle';
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
```

Comment:
[CODE KNOWLEDGE] - There's no need to manually assign the values of the constructor parameters to class properties since TypeScript does this automatically when you use an access modifier like `public` in the constructor parameter. You can simplify the constructor body.